frontend/public/components/catalog/catalog-item-icon.tsx

```diff
@@ -100,6 +100,8 @@ import * as wordpressImg from '../../imgs/logos/wordpress.svg';
 import * as xamarinImg from '../../imgs/logos/xamarin.svg';
 import * as zendImg from '../../imgs/logos/zend.svg';
 import * as operatorImg from '../../imgs/logos/operator.svg';
+import * as vscodeImg from '../../imgs/vscode.png';
+import * as intellijImg from '../../imgs/intellij.png';
```

Comment:
The other logos was part of the logos folder. I'm also unsure if we want add these both to the catalog list.. Its not wrong, but also not necessary.